projects/spectator/src/lib/matchers.ts

```diff
@@ -65,7 +65,7 @@ const hasCss = (el: HTMLElement, css: { [key: string]: string }) => {
         continue;
       }
 
-      if (trim($el.css(prop)) !== trim(value) && trim(el.style[prop]) !== trim(value)) {
+      if (trim($el.get(0).style[prop]) !== trim(value) && trim(el.style[prop]) !== trim(value)) {
```

Comment:
this is the breaking thing!
it stopped working for styles from .css/.scss/.styl/etc files!

can we revert that?
it's not a fix for types it's change for the logic !

Reply:
@dojchek could u pls fix that back

Reply:
@dojchek please handle this. You said this isn't a breaking change.

Reply:
@NetanelBasal I will revert it. I didn't say it's not a breaking change, but I surely would not push it if I knew it was..
@Coffee-Tea Can you please provide the tests which makes this fail?
Sorry for not seeing this earlier.

Reply:
I see that it has been already reverted.